stripper/ze_backrooms_deathbed_t1f.cfg

```diff
@@ -0,0 +1,134 @@
+;workaround for missing buyzone
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+	}
+	insert:
+	{
+		"OnMapSpawn" "ConsoleCommandmp_buy_anywhere 10-1"
+		"OnMapSpawn" "ConsoleCommandmp_buy_anywhere 018-1"
+	}
+}
+
+;patch tp avoidance spot in spawn
+add:
+{
+	"classname" "trigger_push"
+	"targetname" "antidelay_push"
+	"speed" "400"
+	"pushdir" "0 -180 0"
+	"origin" "2333 -187 2658.5"
+	"model" "*85"
+	"spawnflags" "4097"
+	"StartDisabled" "1"
+}
```

Comment:
Fix the TP's, workarounds are not ideal, and exploitable in this case (explained in Discord).